pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -177,8 +177,15 @@ func (r *ReconcileOperConfig) Reconcile(request reconcile.Request) (reconcile.Re
 		}
 	}
 
+	// Bootstrap any resources
+	bootstrapResult, err := network.Bootstrap(&operConfig.Spec, r.client)
+	if err != nil {
+		log.Printf("Failed to bootstrap: %v", err)
```

Comment:
You should have a SetDegraded call here. Something like
`r.status.SetDegraded(..., "BootstrapError", ...)`

Reply:
Also, for user-facing error messages, let's use "reconcile platform networking resources" rather than "bootstrap" as the terminology here.